File: JetpackPratice/app/src/main/java/com/sword/jetpackpratice/MainActivity.java
package com.sword.jetpackpratice;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if(savedInstanceState == null) {
            ProductFragment fragment = new ProductFragment();

            getSupportFragmentManager().beginTransaction().
                    add(R.id.fragment_container, fragment, ProductFragment.TAG).commit();
        }
    }
}

File: SwordDemoLibrary/hencoder/src/main/java/com/httptheory/MainActivity.kt
package com.sword.httptheory

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.httptheory.okhttp.OkHttpSample

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //RetrofitNetUtil().requestByRetrofit()
        OkHttpSample.testRequest()
    }
}